group-coordinator/src/main/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntime.java

```diff
@@ -1359,7 +1359,8 @@ public void scheduleUnloadOperation(
         log.info("Scheduling unloading of metadata for {} with epoch {}", tp, partitionEpoch);
 
         scheduleInternalOperation("UnloadCoordinator(tp=" + tp + ", epoch=" + partitionEpoch + ")", tp, () -> {
-            withContextOrThrow(tp, context -> {
+            CoordinatorContext context = coordinators.get(tp);
+            if (context != null) {
                 if (context.epoch < partitionEpoch) {
```

Comment:
You need to look the context before you use it.

Reply:
ah my bad! thankss I'll add it in